recipe/meta.yaml

```diff
@@ -1,22 +1,20 @@
 # keep this without major version to let the bot pick it up
-{% set version = "21.12" %}
-# protobuf doesn't add the major version in the tag...
+{% set version = "22.3" %}
+# protobuf doesn't add the major version in the tag, it's defined per language in
+# https://github.com/protocolbuffers/protobuf/blob/main/version.json
 {% set major = "4" %}
-# libprotobuf now has a different major version than protobuf
-{% set lib_major = "3" %}
```

Comment:
@h-vetinari, I think we want to keep this and just update it to 4.  There's no guarantee that the c++ and python version numbers will match in the future, right?

Reply:
If they ever diverge again, we introduce the variable again. Not point keeping it around if that's not the case IMO.

Reply:
I disagree. I think the safer thing to do would be to keep `lib_major` as a reminder that the c++ and python version major versions may differ.  This has caused significant confusion and may do so in the future.

Reply:
What's in this recipe or not will not reduce confusion about protobuf's unique versioning scheme and it's our job as maintainers to understand the pieces involved.

Also, this cycle of abseil/grpc/protobuf updates is exceptionally painful to pull off, so coming into a WIP PR with a big red "requested changes" based on a cosmetic (= not functionally relevant) preference is really... detrimental to me feeling like it's worth my free time to try to untangle this mess.

Reply:
I really do appreciate the work you put into this. It would help me as a maintainer if we keep the separate major version numbers but I will let it be if it's causing you anxiety.